services/maven-central/maven-central.service.js

```diff
@@ -52,7 +52,9 @@ module.exports = class MavenCentral extends BaseXmlService {
     },
   ]
 
-  static defaultBadgeData = { label: 'maven-central' }
+  static defaultBadgeData = {
+    label: 'maven-central',
+  }
```

Comment:
Out of curiosity, why this change?

Reply:
@calebcartwright Just to align the object format as I have done in all files.

Reply:
prettier will automatically wrap lines when they need to be, though generally respects the original line wrapping as well. This one is really short so it could easily fit on one line. 

No need to revert this change, but also don't feel obligated to wrap these really simple/short object literals on multiple lines either. In some cases, such as larger route definitions, they're more readable, but otherwise keeping short fields like this one line helps make the files ever so smaller :smile:

Reply:
Okay @calebcartwright  understood thank you  😄